Client/Frontend/Browser/BrowserLocationView.swift

```diff
@@ -50,6 +50,8 @@ class BrowserLocationView : UIView, UIGestureRecognizerDelegate {
         stopReloadButton = UIButton()
         stopReloadButton.setImage(ImageReload, forState: .Normal)
         stopReloadButton.addTarget(self, action: "SELtapStopReload", forControlEvents: .TouchUpInside)
+        stopReloadButton.isAccessibilityElement = true
+        stopReloadButton.accessibilityLabel = NSLocalizedString("Reload", comment: "Accessibility label for the reload button")
```

Comment:
Note that `stopReloadButton` is actually a combined button for stopping and reloading, so just "Reload" isn't sufficient. Can you move this label down to the `loading` property, where we set an image when it changes state? Then you can make the label "stop" when it's a stop button, and "reload" when it's a reload button.